### PR TITLE
feat: Update the llama-forge-template

### PR DESCRIPTION
**Pull request by efecarranza**
## Motivation:

We've made some optimizations to how we work at Llama and thus, have decided to update our template to reflect these changes.

## Modifications:

Remove JavaScript dependencies.
Remove Makefile.
Use `justfile`.
Remove prettier.
Update foundr.toml
Update README.md
Move tests to separate folder on root.

## Result:

New structure for llama-forge-template.

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -6,8 +6,20 @@ optimizer = true
 optimizer_runs = 200
 libs = ['lib']
 src = 'src'
-test = 'test'
+test = 'tests'
 out = 'out'
+bytecode_hash = 'none'
```
Comment (0xrajath) [inline foundry.toml]: You could remove this. It was very specific to Vertex

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [pull_request]
+
+env:
+  RPC_URL: ${{ secrets.RPC_URL }}
+  BLOCK_NUMBER: ${{ secrets.BLOCK_NUMBER }}
```
Comment (0xrajath) [inline .github/workflows/ci.yml]: You can remove the block number here as well. We can always fork in the test

**Review by 0xrajath — APPROVED** [review]
LGTM. Left a couple of comments

File: foundry.toml (hunk 1 above)
Comment (efecarranza) [reply foundry.toml]: Yeah, I thought about it but at the same time maybe there's conflicts with address generation in the future, so kinda torn hah.

**Pull request merged**